packages/database/src/models/SyncSession.ts

```diff
@@ -41,7 +42,11 @@ export class SyncSession extends Model {
   }
 
   static async addDebugInfo(id: string, info: object) {
-    const session = await this.findOne({ where: { id } });
+    // the SKIP LOCKED means we don't lock the sync_sessions table's row
+    // if it's already contended. but that also means that we might lose
+    // some debug info in some cases. so let's debug log it as well.
+    log.debug('Sync session debug', { id, ...info });
```

Comment:
Unfortunately `debugInfo` is now a misnomer, it will soon be renamed `info` and is used for actual business rules 

We could split out `addDebugInfo` and `addInfo` utils, which can write to the same column but with debug level allowing skips?

Reply:
Done, but I don't know which are critical and which aren't, so I've changed one but left the others as debug for now.